### DISCUSSION TITLE
Position window onFocus instead of using keyboard shortcuts

### DISCUSSION THREAD

**Discussion by jasongekeler** [rxhanson/Rectangle#593] [Ideas]
Is there any way to position a window when that window is focused and return it to its previous position when it loses focus? here is my senario...

I have a 49" monitor. I lay all my windows out the way I like them, but some are just too far away on account of the monitor size. I'd love to be able to ALT/TAB to select the app i want to use and have that apps window use the "center third" configuration. Then when alt tab to another window, the previous one moves back into the position and the new one is not "center third".

Really like Rectangle and would love to see it handle this senario as well.

**Comment by rxhanson** [comment]
I like this idea. Off the top of my head, there are a few things that might stand in the way from it being ideal, like determining when applications are switched via the keyboard rather than a click (Rectangle doesn't listen to keystrokes - just registers shortcuts), and how this would feel with apps that have more than one window. When I get some more time, I'll come back to this.
